Package.resolved

```diff
@@ -253,6 +253,15 @@
           "version": "3.3.0"
         }
       },
+      {
+        "package": "VaporMonitoring",
+        "repositoryURL": "https://github.com/Yasumoto/VaporMonitoring.git",
```

Comment:
[Non-blocking comment] How do you plan to perform syncs / updates of this repo?

Reply:
Once we get the upstream PR merged (at https://github.com/vapor-community/VaporMonitoring/pull/5) then we'll revert back to upstream.

Reply:
Once we get the upstream PR merged (at vapor-community/VaporMonitoring#5) then we'll revert back to upstream.